pages/installation/operating_system_packages.rst

```diff
@@ -64,7 +64,9 @@ Debian 8, Ubuntu 16.06 systemd     ``sudo systemctl enable graylog-server``
 Update to latest Version
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you like to install the latest Graylog Version as Update and you had used the repository package before just install the latest repository package and use your distribution command to update the ``graylog-server`` package::
+When you had used the repository package before to install Graylog, you need to update the repository package at first. The new repository package will update and replace the repository URL. Without this update you will only get Bugfix releases of your installed Version.
+
+The update will work like a fresh installation::
 
   $ wget https://packages.graylog2.org/repo/packages/graylog-2.1-repository_latest.deb
```

Comment:
It will update & replace:

```
# dpkg --info graylog-2.1-repository_latest.deb
 new debian package, version 2.0.
 size 2044 bytes: control archive=594 bytes.
      80 bytes,     2 lines      conffiles
     585 bytes,    12 lines      control
     230 bytes,     3 lines      md5sums
 Package: graylog-2.1-repository
 Version: 1-3
 License: GPLv3
 Vendor: graylog
 Architecture: all
 Maintainer: Graylog, Inc. <hello@graylog.org>
 Installed-Size: 0
 Replaces: graylog2-0.92-repository-ubuntu14.04, graylog2-0.91-repository-ubuntu14.04, graylog2-0.90-repository-ubuntu14.04, graylog-1.0-repository-ubuntu14.04, graylog-1.1-repository-ubuntu14.04, graylog-1.2-repository-ubuntu14.04, graylog-1.3-repository-ubuntu14.04, graylog-2.0-repository
 Section: optional
 Priority: extra
 Homepage: https://www.graylog.org/
 Description: Package to install Graylog 2.1 GPG key and repository```